src/Sylius/Component/Addressing/Tests/Checker/CountryProvincesDeletionCheckerTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\Component\Addressing\Checker;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Sylius\Component\Addressing\Checker\CountryProvincesDeletionChecker;
+use Sylius\Component\Addressing\Checker\CountryProvincesDeletionCheckerInterface;
+use Sylius\Component\Addressing\Model\CountryInterface;
+use Sylius\Component\Addressing\Model\ProvinceInterface;
+use Sylius\Component\Addressing\Model\ZoneMemberInterface;
+use Sylius\Resource\Doctrine\Persistence\RepositoryInterface;
+
+final class CountryProvincesDeletionCheckerTest extends TestCase
+{
+    /** @var RepositoryInterface<ZoneMemberInterface>&MockObject */
+    private MockObject $zoneMemberRepositoryMock;
+
+    /** @var RepositoryInterface<ProvinceInterface>&MockObject */
+    private MockObject $provinceRepositoryMock;
```

Comment:
```suggestion
    private MockObject&RepositoryInterface $zoneMemberRepository;

    private MockObject&RepositoryInterface $provinceRepository;
```
Wdyt? But it is not a blocker of course 🖖🏻